src/Components/Programs/Timeline.js

```diff
@@ -0,0 +1,99 @@
+import React, {useEffect} from 'react';
+import {ScrollView,Text,View,TouchableWithoutFeedback} from 'react-native';
+import {Event,Line,Date,Marker,Fade,Stroke} from './styles';
+import $ from 'jquery';
+import {ArrowLeftOutlined ,ArrowRightOutlined} from '@ant-design/icons'
+import dayjs from 'dayjs'
+const events=[
```

Comment:
this should be kept in content js so we can edit

Reply:
okay